visidata/loaders/archive.py

```diff
@@ -16,9 +16,19 @@ class ZipSheet(Sheet):
         ColumnAttr('compress_size', type=int)
     ]
 
+    def open_zipfile(self, fp, *args, **kwargs):
+        '''Use VisiData input to handle password-protected zip files.'''
+        try:
+            return fp.open(*args, **kwargs)
+        except RuntimeError as err:
+            if 'password required' in err.args[0]:
+                pwd = vd.input(f'{args[0].filename} is encrypted, enter password: ', display=False)
+                return fp.open(*args, **kwargs, pwd=pwd.encode())
```

Comment:
Let's make `'utf-8'` an explict parameter to `encode()`.  (It's the default option but would be helpful I think, especially given that this is a password.)